### PR TITLE
Decision server basic ssl

### PR DESCRIPTION
**Pull request by spolti**

### CODE REVIEW

File: decisionserver/pom.xml [hunk 1]
```diff
@@ -45,7 +45,6 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
-                    <runOrder>alphabetical</runOrder>
```
Comment (alesj) [inline decisionserver/pom.xml]: Lets just leave this there, so we're deterministic.

File: decisionserver/src/test/java/org/jboss/test/arquillian/ce/decisionserver/DecisionServerAmqTest.java [hunk 2]
```diff
@@ -72,8 +72,7 @@
 @RunWith(Arquillian.class)
 @RunInPod
 @ExternalDeployment
-//@Template(url = "https://raw.githubusercontent.com/jboss-openshift/application-templates/master/decisionserver/decisionserver62-basic-s2i.json",
-@Template(url = "https://raw.githubusercontent.com/spolti/application-templates/CLOUD-435/decisionserver/decisionserver62-amq-s2i.json",
+@Template(url = "https://raw.githubusercontent.com/jboss-openshift/application-templates/master/decisionserver/decisionserver62-amq-s2i.json",
```
Comment (alesj) [inline decisionserver/src/test/java/org/jboss/test/arquillian/ce/decisionserver/DecisionServerAmqTest.java]: So this template is OK now?

File: decisionserver/src/test/java/org/jboss/test/arquillian/ce/decisionserver/DecisionServerBasicTestSecure.java [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+                @TemplateParameter(name = "KIE_SERVER_USER", value = "${kie.username:kieserver}"),
+                @TemplateParameter(name = "KIE_SERVER_PASSWORD", value = "${kie.password:Redhat@123}")
+        }
+)
+@OpenShiftResources({
+        @OpenShiftResource("https://raw.githubusercontent.com/jboss-openshift/application-templates/master/jboss-image-streams.json"),
+        @OpenShiftResource("classpath:decisionserver-service-account.json"),
+        @OpenShiftResource("classpath:decisionserver-app-secret.json")
+})
+public class DecisionServerBasicTestSecure {
```
Comment (alesj) [inline decisionserver/src/test/java/org/jboss/test/arquillian/ce/decisionserver/DecisionServerBasicTestSecure.java]: The name is wrong --- it must end with Test!

**Comment by alesj** [comment]
@spolti please rebase with current upstream + remove any dup code -- as I did in previous PR.
(as I see this test is 99% same as Basic test ...)

**Comment by spolti** [comment]
Yes, it is basically the same, only the url was changed to use https.

**Comment by alesj** [comment]
@spolti I get this ...

Caused by: java.lang.IllegalStateException: Checker [(Required pods: 1)] failed to pass.
	at org.jboss.arquillian.ce.utils.Containers.delay(Containers.java:52)

**Comment by alesj** [comment]
Merged.

@spolti but do check on why it fails (for me)

Plus, in the future PRs make sure you reduce your duplicate code -- no need to copy/paste all over the place. ;-)

**Comment by spolti** [comment]
Thanks @alesj, I am doing that, today I'll send another commit with some code reductions.

**Pull request closed without merging**